src/components/sidebar/sidebar.jsx

```diff
@@ -0,0 +1,264 @@
+import "./sidebar.css";
+import { useProducts } from "contexts/product-context";
+
+function SortComponent() {
+  const { state, dispatch } = useProducts();
```

Comment:
names could've been more specific, also destructure the state if you're repeating the variable multiple times

Reply:
Cool. Suggestion taken.